src/components/Modal/Modal.scss

```diff
@@ -0,0 +1,31 @@
+.modal__wrapper {
```

Comment:
Since we are using `SCSS` and I suppose you are using BEM for writing CSS.classes. I will encourage you to modify the styles something like this:
```css
.model {
  &__wrapper {
   /* Rules go here */
  }
  &__backdrop { {
   /* Rules go here */
  }
  &__box { {
   /* Rules go here */
  }
}
```